addons/l10n_in_gstin_status/models/res_partner.py

```diff
@@ -0,0 +1,106 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+import re
+
+from odoo import api, fields, models, _
+from odoo.addons.iap import jsonrpc
+from odoo.exceptions import UserError, AccessError, ValidationError
+from odoo.addons.l10n_in.const import DEFAULT_IAP_ENDPOINT, DEFAULT_IAP_TEST_ENDPOINT
+
+_logger = logging.getLogger(__name__)
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    l10n_in_gstin_verified_status = fields.Boolean(
+        string="GST Status",
+        compute="_compute_l10n_in_gst_status",
+        store=True,
+        tracking=True,
+    )
+    l10n_in_gstin_verified_date = fields.Date(
+        string="GSTIN Verified Date",
+        compute="_compute_l10n_in_gst_status",
+        store=True,
+        tracking=True,
+    )
+
+    @api.depends('vat')
+    def _compute_l10n_in_gst_status(self):
+        """
+        Reset GST Status Whenever the `vat` of partner changes
+        """
+        for partner in self:
+            if partner.country_code == 'IN':
+                partner.l10n_in_gstin_verified_status = False
+                partner.l10n_in_gstin_verified_date = False
+
+    def _l10n_in_gstin_status_get_url(self, company):
+        if company.sudo().l10n_in_edi_production_env:
+            default_endpoint = DEFAULT_IAP_ENDPOINT
+        else:
+            default_endpoint = DEFAULT_IAP_TEST_ENDPOINT
+        endpoint = self.env["ir.config_parameter"].sudo().get_param("l10n_in_gstin_status.endpoint", default_endpoint)
```

Comment:
a different parameter here as well?  Apart from the /iap/l10n_in_reports/1/public/search, this method could be in l10n_in

Reply:
> a different parameter here as well? Apart from the /iap/l10n_in_reports/1/public/search, this method could be in l10n_in

I did not get this comment rest have been applied